src/Benchmarks/Startup.cs

```diff
@@ -154,6 +154,12 @@ public void ConfigureServices(IServiceCollection services)
             {
                 services.AddResponseCaching();
             }
+
+            if (Scenarios.Any("SignalR"))
```

Comment:
I am fine having them in Benchmarks, but you might prefer a simpler custom application to have more flexibility. Even in this repository.